snmplib/snmp_api.c

```diff
@@ -7140,7 +7144,7 @@ netsnmp_oid_find_prefix(const oid * in_name1, size_t len1,
     min_size = SNMP_MIN(len1, len2);
     for(i = 0; i < (int)min_size; i++) {
         if (in_name1[i] != in_name2[i])
-            return i;    /* '�' is the first differing subidentifier
+            return i;    /* '�' is the first differing subidentifier
```

Comment:
What is the goal of this change?  What is the new character inside the quotes?  Github can't render it.